packages/extensions/openapi-to-typespec/src/generate/generate-service-information.ts

```diff
@@ -20,9 +22,18 @@ export function generateServiceInformation(program: TypespecProgram) {
   }
 
   if (isArm && serviceInformation.armCommonTypeVersion) {
-    definitions.push(
-      `@armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.${serviceInformation.armCommonTypeVersion})`,
-    );
+    if (VALID_VERSION.includes(serviceInformation.armCommonTypeVersion)) {
+      definitions.push(
+        `@armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.${serviceInformation.armCommonTypeVersion})`,
+      );
+    } else {
+      definitions.push(
+        `// FIXME: Common type version ${serviceInformation.armCommonTypeVersion} is not supportted for now.`,
```

Comment:
supported.

Any why do we need this? 
 '// @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.${serviceInformation.armCommonTypeVersion})'

Reply:
this version is used when converting back to swagger.